src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -418,7 +418,18 @@ def file_issue(testcase,
     issue.components.add(policy.unreproducible_component)
 
   issue.reporter = user_email
-  issue.save()
+
+  try:
+    issue.save()
+  except:
+    if policy.fallback_component:
+      issue.components.clear()
+      issue.components.add(policy.fallback_component)
+      if policy.fallback_policy_message:
+        message = policy.fallback_policy_message.format(
+            ' '.join(metadata_components))
```

Comment:
Rather than using .format(), can you replace `"%COMPONENTS%"` instead? '{}' is not descriptive and not extensible if we want to add more substitutions.

Reply:
Got it! Done.